packages/config-schema/schema.json

```diff
@@ -0,0 +1,335 @@
+              "additionalProperties": false,
+              "properties": { "enabled": { "type": "boolean" } }
+            },
+            "min_length": {
+              "type": "integer"
+            }
+          }
+        },
+        "provider": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "apple": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": { "type": "boolean" },
+                "client_id": { "type": "string" },
+                "team_id": { "type": "string" },
+                "key_id": { "type": "string" },
+                "private_key": { "type": "string" },
+                "scope": { "$ref": "#/$defs/comma_list" }
+              }
+            },
+            "github": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": { "type": "boolean" },
+                "client_id": { "type": "string" },
+                "client_secret": { "type": "string" },
+                "authorization_url": { "type": "string" },
+                "token_url": { "type": "string" },
+                "user_profile_url": { "type": "string" },
+                "scope": { "$ref": "#/$defs/comma_list" }
+              }
+            },
+            "bitbucket": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": { "type": "boolean" },
+                "client_id": { "type": "string" },
+                "client_secret": { "type": "string" }
+              }
+            },
+            "facebook": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": { "type": "boolean" },
+                "client_id": { "type": "string" },
+                "client_secret": { "type": "string" },
+                "profile_fields": { "$ref": "#/$defs/comma_list" },
+                "scope": { "$ref": "#/$defs/comma_list" }
+              }
+            },
+            "gitlab": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": { "type": "boolean" },
+                "client_id": { "type": "string" },
+                "client_secret": { "type": "string" },
+                "base_url": { "type": "string" },
+                "scope": { "$ref": "#/$defs/comma_list" }
+              }
+            },
+            "twilio": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": { "type": "boolean" },
+                "account_sid": { "type": "string" },
+                "auth_token": { "type": "string" },
+                "messaging_service_id": { "type": "string" }
+              }
+            },
```

Comment:
I think Twilio here is a leftover

```suggestion
```

Reply:
yes, thanks for that Asen. This PR needs some refactoring - we will probably move to `zod` to get typings and transformers, in addition to a generated JSON schema